src/support/rgw_support_bundle

```diff
@@ -39,32 +42,33 @@ class SupportBundleError(BaseError):
 
 class RGWSupportBundle:
     """Generate rgw specific support bundle."""
-    _ceph_conf_file = '/etc/ceph/ceph.conf'
-    _ceph_crashdump_dir = '/var/log/ceph/crash'
     _tmp_src = '/tmp/rgw/'
 
     @staticmethod
     def generate(bundle_id: str, target_path: str, cluster_conf: str, **filters):
         """ Generate a tar file. """
         coredumps = filters.get('coredumps', False)
-        cortx_config_store = MappedConf(cluster_conf)
-        log_base = cortx_config_store.get('cortx>common>storage>log')
         machine_id = Conf.machine_id
+        cortx_config_store = MappedConf(cluster_conf)
+        log_base = cortx_config_store.get(LOG_PATH_KEY)
+        config_path = cortx_config_store.get(CONFIG_PATH_KEY)
+        rgw_config_dir = os.path.join(config_path, COMPONENT_NAME, machine_id)
+        rgw_config_file = os.path.join(rgw_config_dir, RGW_CONF_FILE)
 
         if os.path.exists(RGWSupportBundle._tmp_src):
             RGWSupportBundle._cleanup()
         os.makedirs(RGWSupportBundle._tmp_src, exist_ok=True)
         # copy configuration files
         try:
-            shutil.copyfile(RGWSupportBundle._ceph_conf_file,\
-                os.path.join(RGWSupportBundle._tmp_src, 'ceph.conf'))
+            shutil.copyfile(rgw_config_file,\
+                os.path.join(RGWSupportBundle._tmp_src, RGW_CONF_FILE))
         except FileNotFoundError as fe:
             raise SupportBundleError(errno.ENOENT, "File not found %s", fe)
 
         # copy rgw client log files
         log_dir = os.path.join(log_base, f'rgw/{machine_id}')
         if os.path.exists(log_dir):
-            regex = re.compile('(rgw.client.*log)')
+            regex = re.compile('(rgw_setup.*log)')
```

Comment:
Mini Provisioner rgw logs are getting generated with filename: rgw_setup.log, so changed the filename from rgw.client to rgw_setup
Cc @sachitanands

Reply:
these are two different logs .
1) logs generated during mini-provisioner should go to rgw_setup.log (Name can be change to something rgw_deployemnt.log )
2) actual rgw service logs should go to rgw.client.log which will be having all rgw request logs.

Hence we need these two logs separately.

Reply:
Got it.
Added both the log files.